blocksuite/affine/components/src/rich-text/format/config.ts

```diff
@@ -124,5 +125,14 @@ export const textFormatConfigs: TextFormatConfig[] = [
     action: host => {
       host.std.command.chain().pipe(toggleLink).run();
     },
+    // should check text length
+    checkText: host => {
```

Comment:
normally, we call it textChecker or textValidator.

Reply:
ok